src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,20 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int bucketCapacity;
```

Comment:
The field `bucketCapacity` should have an access modifier. Consider using private to encapsulate the data, and provide public getters and setters for access.